src/util/types.ts

```diff
@@ -1034,6 +1036,7 @@ export interface CommonTooltipOption<FormatterParams> {
     borderColor?: ColorString
     borderRadius?: number
     borderWidth?: number
+    boxShadow?: ColorString
```

Comment:
As commented above, should not be named as `boxShadow` (and the ts type of css boxShadow is not `ColorString`).
Should better be: 
shadowBlur
shadowColor
shadowOffsetX
shadowOffsetY